packages/block-library/src/cover-image/index.js

```diff
@@ -54,12 +55,14 @@ const blockAttributes = {
 
 export const name = 'core/cover-image';
 
+const icon = <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 4h7V2H4c-1.1 0-2 .9-2 2v7h2V4zm6 9l-4 5h12l-3-4-2.03 2.71L10 13zm7-4.5c0-.83-.67-1.5-1.5-1.5S14 7.67 14 8.5s.67 1.5 1.5 1.5S17 9.33 17 8.5zM20 2h-7v2h7v7h2V4c0-1.1-.9-2-2-2zm0 18h-7v2h7c1.1 0 2-.9 2-2v-7h-2v7zM4 13H2v7c0 1.1.9 2 2 2h7v-2H4v-7z" /><path d="M0 0h24v24H0z" fill="none" /></svg>;
```

Comment:
@ZebulanStanphill I think you missed flipping this one back :)

Reply:
@chrisvanpatten No, that was intentional, because since `cover-image` does not have a separate `edit.js`, I was not sure if I could export `settings.icon` and then reference it from within the definition of `settings` itself. Actually, `cover-image` was part of the reason why I did my initial approach.

Reply:
I considered splitting the `edit` function of `cover-image` into a separate file, but I decided that was out-of-scope for this PR. What do you think I should do?

Reply:
@ZebulanStanphill Ah, I see now. I think it makes sense to split it out but it might be worth another reviewer jumping in to give this a sanity check before going too far down that path.

Reply:
@chrisvanpatten This PR has some merge conflicts currently. I have created a PR to split the Cover Image `edit` function into its file: #10582. Once that is merged, I'll rebase or replace this PR.